### DISCUSSION TITLE
Concerning GPU-accelerated plotting

### DISCUSSION THREAD

**Discussion by GTANAdam** [madMAx43v3r/chia-plotter#237] [General]
Yesterday, I saw an [issue](https://github.com/madMAx43v3r/chia-plotter/issues/184) where the poster was requesting the developer's contact details, stating that they've got what Chia's been missing the whole time, a "GPU-accelerated" plotter, but it seems that the message now has been redacted for unknown reasons but if I may, I want to boast my opinion concerning this. I sincerely do not think that a GPU accelerated plotter is a good idea for the ecosystem right now, it will basically make the rich richer and the poor poorer, which will basically change the whole game and kill Chia's capital due to high liquidity, it's what miners do, they leech. it will also make GPUs much more harder to find for those who wish to do productive work in CAD or other researching and whatnot. just my 1 cent concerning GPU plotting. thank you.

**Comment by madMAx43v3r** [comment]
I talked to him, he wanted to sell me his "GPU plotter", but based on what he was claiming I concluded it's most likely a scam

**Reply by SebMoore** [reply]
Of course he did - oh well 🤣
